rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -69,6 +69,10 @@ docker run --name $docker_name \
   --security-opt apparmor:unconfined \
   --volume /var/drivers/nvidia/current:/usr/local/nvidia:ro \
   --volume /tmp/pai-root/alive/$APP_ID:/alive \
+  --label PAI_HOSTNAME="$(hostname)" \
+  --label PAI_JOB_NAME={{{ jobData.jobName }}} \
+  --label PAI_USERNAME={{{ jobData.username }}} \
+  --label PAI_TASK_ROLE_NAME={{{ taskData.name }}} \
   --env APP_ID=$APP_ID \
```

Comment:
Please add label for "•	pai_task_index_in_role: Task index in role".

Reply:
Delay to metrics implementation stage.

Reply:
ok